### PR TITLE
Explicit methods for geometry and separate methods for doc objects

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -47,7 +53,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Changed `compas.geometry.curves.nurbs.NurbsCurve.__new__` to prevent instantiation of `NurbsCurve` directly.
 * Changed `compas_rhino.geometry.curves.new_nurbscurve_from_...` to `nurbscurve_from_...`.
 * Fixed `compas_ghpython` Grasshopper components not included in published pakcage.
-* Chnaged `compas.colors.Color.coerce` to take color as is, if it is already aninstance of `compas.colors.Color`.
+* Changed `compas.colors.Color.coerce` to take color as is, if it is already aninstance of `compas.colors.Color`.
```
Comment (chenkasirer) [inline CHANGELOG.md]:
```suggestion
* Changed `compas.colors.Color.coerce` to take color as is, if it is already an instance of `compas.colors.Color`.
```

File: src/compas_rhino/conversions/surfaces.py [hunk 2]
```diff
@@ -295,61 +71,33 @@ def surface_to_compas_quadmesh(surface, nu, nv=None, weld=False, facefilter=None
     weld: bool, optional
         Weld the vertices of the mesh.
         Default is False.
-    facefilter: callable, optional
-        A filter for selection which Brep faces to include.
-        If provided, the filter should return True or False per face.
-        A very simple filter that includes all faces is ``def facefilter(face): return True``.
-        Default parameter value is None in which case all faces are included.
-    cls: :class:`compas.geometry.Mesh`, optional
+    cls: :class:`compas.datastructures.Mesh`, optional
```
Comment (chenkasirer) [inline src/compas_rhino/conversions/surfaces.py:71]: what is this for?

File: src/compas_rhino/conversions/surfaces.py [hunk 3]
```diff
@@ -2,21 +2,15 @@
...
-import Rhino  # type: ignore
-
+# import Rhino  # type: ignore
 from compas.datastructures import Mesh
 from compas.geometry import NurbsSurface
-from compas.geometry import Point
-from compas.tolerance import TOL
+from compas.geometry import Surface
+
+# from compas.tolerance import TOL
```
Comment (chenkasirer) [inline src/compas_rhino/conversions/surfaces.py]: remove?

File: src/compas_rhino/conversions/surfaces.py [hunk 4]
```diff
@@ -2,21 +2,15 @@
 from __future__ import division
 from __future__ import print_function
 
-import Rhino  # type: ignore
-
+# import Rhino  # type: ignore
```
Comment (chenkasirer) [inline src/compas_rhino/conversions/surfaces.py]: remove?

File: src/compas_rhino/objects.py [hunk 5]
```diff
@@ -967,175 +996,175 @@ def select_meshes(message="Select multiple meshes."):
...
-    mobj = sc.doc.Objects.Find(guid)
-    mgeo = mobj.Geometry
-    temp = mgeo.TopologyEdges.GetTopologyVertices(eindex)
-    tvindices = temp.I, temp.J
-    vindices = []
-    for tvindex in tvindices:
-        temp = mgeo.TopologyVertices.MeshVertexIndices(tvindex)
-        vindices.append(temp[0])
-    return vindices
+# def get_meshes(layer=None):
```
Comment (chenkasirer) [inline src/compas_rhino/objects.py]: remove?

File: src/compas_rhino/conversions/surfaces.py (hunk 2 above)
Comment (chenkasirer) [reply src/compas_rhino/conversions/surfaces.py:71]: ah is this so you can get e.g. a `Polyhedron` instead?

**Review by chenkasirer — APPROVED** [review]
left some comments but generally lgtm

File: src/compas_rhino/conversions/surfaces.py (hunk 2 above)
Comment (tomvanmele) [reply src/compas_rhino/conversions/surfaces.py:71]: no, if you want a subclass of mesh instead of the default

**Pull request merged**